src/app/configurator/configurator.component.scss

```diff
@@ -0,0 +1,46 @@
+$line-color: rgb(141, 140, 140);
+
+.margin-top {
+  margin-top: 100px;
+}
+.vertical-line {
+  width: 1px;
+  background-color: $line-color;
+  height: 100%;
+  margin: 0 auto;
+}
+.mat-dialog-actions {
+  margin-bottom: 0;
+}
+.leftt {
+  justify-content: flex-start !important;
```

Comment:
Please don't use !important. Also there's a type in the class name

Reply:
sorry, I know. I don't use this class. I removed it